code/modules/uplink/uplink_items/explosive.dm

```diff
@@ -92,12 +92,16 @@
 
 /datum/uplink_item/explosives/syndicate_bomb
 	name = "Syndicate Bomb"
-	desc = "The Syndicate bomb is a fearsome device capable of massive destruction. It has an adjustable timer, \
-			with a minimum of 60 seconds, and can be bolted to the floor with a wrench to prevent \
+	desc = null
+	progression_minimum = 40 MINUTES
+	item = /obj/item/sbeacondrop/bomb
+	cost = 11
+
+/datum/uplink_item/explosives/syndicate_bomb/New()
+	. = ..()
+	desc ||= "The Syndicate bomb is a fearsome device capable of massive destruction. It has an adjustable timer, \
```

Comment:
Just an alternative idea, instead of adding the description in New, maybe you can use ``replacetext()`` for %BOMB_TIMER or something.